src/common/i18n/__init__.py

```diff
@@ -155,6 +155,9 @@ def set_locale(locale: str) -> bool:
     )
     + os.sep
 )
+_jinja_pattern: str = (
+    os.sep + os.sep.join(['venv', 'Lib', 'site-packages', 'jinja2']) + os.sep
```

Comment:
Please remove the first part (my venv is at .venv-3.13).
```suggestion
    os.sep + os.sep.join(['Lib', 'site-packages', 'jinja2']) + os.sep
```